src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -141,6 +142,8 @@ define(function (require, exports, module) {
 
             this.getFlux().actions.export.updateExportAsset(
                 this.props.document, this.props.layers, this.props.index, props);
+
+            headlights.logEvent("edit", "export-settings-quality", props);
```

Comment:
You can't log objects. This needs to be a string. If nothing else, you can just concat the format and quality strings.